tests/test_component_media.py

```diff
@@ -108,7 +108,7 @@ class TestComponent(AppLvlCompComponent):
 
         self.assertInHTML(
             """
-            <form data-djc-id-a1bc3f method="post">
+            <form data-djc-id-a1bc41 method="post">
```

Comment:
Why are these updating?

Reply:
Before, the `{% component_js_dependencies %}` tags only returned a string. Now they create instances of their own BaseNode subclasses, and return a string at `BaseNode.render()`. And those instances create one ID each [during initialization](https://github.com/EmilStenstrom/django-components/pull/910/files#diff-de799abd623d22009b507f957146f2e4851d4def29c7891fe979e04ce0aeb15dR329).